.buildconfig.yml

```diff
@@ -20,5 +20,5 @@ projects:
     artifactId: glean-gradle-plugin
     publications:
       - name: glean-gradle-plugin
-        type: aar
+        type: jar
```

Comment:
This was wrong to begin with.
a-c also has it as jar: https://github.com/mozilla-mobile/android-components/blob/master/.buildconfig.yml#L435-L439